src/workloads/query/FilterWithComplexLogicalExpression.yml

```diff
@@ -26,753 +26,11 @@ Keywords:
 - insert
 - find
 
-GlobalDefaults:
```

Comment:
Just to confirm- this all moved basically unchanged except for parametrizing collection size to src/phases/query/FilterWithComplexLogicalExpression.yml, yes? Same goes for the other workload?